src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -4858,6 +4858,9 @@ void ManageInsideAdaptiveConvectionAlgo(EnergyPlusData &state, int const SurfNum
     //  TODO: candidate for rework to do zone level calcs once rather than for each surface
     DynamicIntConvSurfaceClassification(state, SurfNum);
 
+    // Set report var after surface has been classified successfully
+    state.dataSurface->SurfIntConvClassificationRpt(SurfNum) = static_cast<int>(state.dataSurface->SurfIntConvClassification(SurfNum));
+
```

Comment:
This looks safe to me. No chance of crash with SurfNum = 0 (would crash all over the place, not just here) and just equating a variable to an int.